src/DistributedLock.Redis/DistributedLock.Redis.csproj

```diff
@@ -49,13 +49,12 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="StackExchange.Redis" Version="2.2.4" />
-    <PackageReference Include="Nullable" Version="1.3.1" Condition="'$(TargetFramework)' != 'netstandard2.1'">
+    <PackageReference Include="StackExchange.Redis" />
+    <PackageReference Include="Nullable" Condition="'$(TargetFramework)' != 'netstandard2.1'">
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All"/>
-    <PackageReference Include="Microsoft.CodeAnalysis.PublicApiAnalyzers" Version="3.3.4" PrivateAssets="All" Condition="$(Configuration) == 'Release'"/>
-    <Using Remove="System.Net.Http"/>
```

Comment:
Fly-by fix: removed this duplicate `Using` item because it appears a few lines below.